libcodechecker/log/option_parser.py

```diff
@@ -140,6 +140,8 @@
 # Unknown options by clang, will be skipped.
 
 UNKNOWN_OPTIONS_MAP_REGEX = {
+    '^-march=armv7-a': 0,  # not supported by clang
```

Comment:
I am a bit uncomfortable removing this argument every time. There are valid clang invocations with this.
E.g. the following works for me:
` clang -no-canonical-prefixes foo.c -target arm-linux-androideabi -march=armv7-a -c`

Reply:
Actually, based on the logs it seems that the target was not detected properly on the first hand. Once that is fixed, probably this armv7-a ignore option can go away. I will update this patch with further fixes.